addons/survey/models/survey_survey.py

```diff
@@ -75,8 +75,8 @@ def default_get(self, fields_list):
     background_image = fields.Image("Background Image")
     background_image_url = fields.Char('Background Url', compute="_compute_background_image_url")
     active = fields.Boolean("Active", default=True)
-    user_id = fields.Many2one(
-        'res.users', string='Responsible',
+    user_ids = fields.Many2many(
```

Comment:
While being there, why not calling it ``restrict_user_ids`` like we do in some addons ? That way it is way easier to grep than "user_ids" :D .